app/tests/test_user.py

```diff
@@ -0,0 +1,179 @@
+import json
+
+from django.test import TestCase
+from rest_framework.test import APIClient
+
+from app.models import User
+from app.serializers import UserSerializer
+
+class UserTestCase(TestCase):
+    fixtures = ('user',)
+    SERVER_NAME = 'http://gucci.com/api'
```

Comment:
![400593_AP00T_1000_001_100_0000_Light-Leather-belt-with-Double-G-buckle](https://user-images.githubusercontent.com/30078143/69970727-69bb1e80-1527-11ea-9d34-b0ca86594081.jpg)

Let's use an appropriate name (we will kill this easter egg just for debugging purposes).